System.Device.I2c/I2cDevice.cs

```diff
@@ -114,26 +127,65 @@ public static I2cDevice Create(I2cConnectionSettings settings)
         /// <param name="settings">Connection settings</param>
         public I2cDevice(I2cConnectionSettings settings)
         {
-            ConnectionSettings = settings;
-            _device = Windows.Devices.I2c.I2cDevice.FromId($"I2C{settings.BusId}", new Windows.Devices.I2c.I2cConnectionSettings(settings.DeviceAddress)
+            _connectionSettings = settings;
+
+            // create the buffer
+            _buffer = new byte[1];
```

Comment:
This is ok, it's just than _buffer should be a SpanByte, this new allocation won't change